components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/factory/CachedJMSConnectionFactory.java

```diff
@@ -72,7 +72,8 @@ public Connection getConnection(String userName, String password) {
         try {
         	connection.start();
         } catch (JMSException e) {
-            logger.error("JMS Exception while starting connection for factory '" + this.connectionFactoryString + "' " + e.getMessage());
+            logger.error("JMS Exception while starting connection for factory '"
+                    + this.connectionFactoryString + "' ", e);
```

Comment:
Connection factory string can have the password ?

Reply:
Will not be an issue here since connectionFactoryString is only the ConnectionFactoryJNDIName (QueueConnectionFactory/TopicConnectionFactory) https://github.com/wso2/carbon-mediation/blob/master/components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/factory/JMSConnectionFactory.java#L127 
and the lookup is actually done at https://github.com/wso2/carbon-mediation/blob/master/components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/factory/JMSConnectionFactory.java#L183?